pkg/client/cache/listers/listers_core.go

```diff
@@ -24,6 +24,7 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/api/v1"
+	"k8s.io/kubernetes/pkg/client/cache"
```

Comment:
Am not a big fan of importing the parent directory. Maybe move listers next to cache?